03/app.test.js

```diff
@@ -1 +1,56 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from "./app";
+
+describe("Random Number", () => {
+  it("should return random number between 1 and 10", () => {
+    const number = randomNumber(1, 10);
+
+    expect(number).toBeGreaterThanOrEqual(1);
+    expect(number).toBeLessThanOrEqual(10);
```

Comment:
Tego typu nie jest jednoznaczny więc nie jest najlepszym rozwiązań.
Jeśli już to napisałbym pętlę, która iteruje od 0 do 100 i sprawdzałbym czy zwrócona wartość jest odpowiednia
czyli `randomNumber(i, i) === i`. Wydajnościowo słabe, ale pewniejsze :)

Reply:
Nie bardzo rozumiem. Mam sprawdzić czy zostaje wylosowana wartość losowa pomiędzy min i max, więc czemu sprawdzać czy zwraca i jesli min i max = i?